src/Email/GlobalSettingValidator.php

```diff
@@ -28,14 +28,18 @@ public function __invoke()
 
     /**
      * @since 2.17.1
+     * @unreleased Only filter value as unique array if set. Some settings do not need to be set, ie donation-receipt_recipient.
      */
     public function validateSetting($value)
     {
-        // Same unique email address for email recipients.
-        $recipientEmails = array_unique(array_filter($value));
+        if( ! empty( $value ) ) {
```

Comment:
@kjohnson Is it logical and expected that this value may be empty? Or is this an incorrect use of this function that ought to be fixed upstream?

Reply:
There is an upstream issue to address, but that would require an outsized refactor. Instead, this restores the empty check that was lost in the refactor.